bin/workflows/pycbc_make_offline_search_workflow

```diff
@@ -57,17 +57,18 @@ def ifo_combos(ifos):
         for ifocomb in combinations:
             yield ifocomb
 
-
-# Log to the screen until we know where the output file is
-logging.basicConfig(format='%(asctime)s:%(levelname)s : %(message)s',
-    level=logging.INFO)
-
```

Comment:
Does this move change anything?

Reply:
Ahh - Ive just seen that there is the second call to the basicConfig to send to the log file

The change to init_logging also makes the default format change to use ISO standard time format including timezone